tests/test_database_types.py

```diff
@@ -58,19 +65,31 @@
             # "bigint", # 8 bytes
         ],
         # https://dev.mysql.com/doc/refman/8.0/en/datetime.html
-        "datetime_no_timezone": ["timestamp(6)", "timestamp(3)", "timestamp(0)", "timestamp", "datetime(6)"],
+        "datetime_no_timezone": [
+            # "timestamp(6)",
+            # "timestamp(3)",
+            # "timestamp(0)",
+            # "timestamp",
+            # "datetime(6)",
+        ],
         # https://dev.mysql.com/doc/refman/8.0/en/numeric-types.html
         "float": [
-            # "float",
-            # "double",
-            # "numeric",
+            "float",
+            "double",
```

Comment:
aWhat about a test where we pass the precision bits to `float`?

Reply:
Maybe? I didn't make sure we're testing every variation.